decidim-elections/app/forms/decidim/votings/check_census_form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # A form to check if data matches census
+    class CheckCensusForm < Form
+      attribute :document_number, String
+      attribute :document_type, String
+      attribute :postal_code, String
+      attribute :day, String
+      attribute :month, String
+      attribute :year, String
+
+      validates :document_number,
+                :document_type,
+                :postal_code,
+                presence: true
+
+      def options_for_document_type_select
+        [
+          I18n.t("passport", scope: "decidim.votings.votings.check_census_form"),
+          I18n.t("dni", scope: "decidim.votings.votings.check_census_form")
```

Comment:
DNI is a Spanish thing, should we just say id_document?

Reply:
💬 This is very tricky, as the list of possible documents used could depend on each installation, or even organization. This is also hardcoded [in the ID verification](https://github.com/decidim/decidim/blob/develop/decidim-verifications/app/forms/decidim/verifications/id_documents/information_form.rb#L10), but in that case it the developers can add a new verification system with different document types if they need it. I'd leave it as it is and would discuss how to allow admins to customize it in the future.